README.md

```diff
@@ -0,0 +1,58 @@
+# Cruise Ship OOP project
+
+## About
+I created this project as part of the [Manchester Codes](https://www.manchestercodes.com/) software engineering course. The objective was to learn about Object Oriented Programming (OOP) principles using JavaScript, and to practise Test Driven Development. 
+
+The project contains three objects that interact with each other, allowing a 'ship' to travel across an itinerary with predetermined ports. 
+
+## Setup
+The project uses JavaScript, Node and Jest. 
+
+Download and initial setup: 
+* Fork the project repository
+* Get the project link, navigate to the local folder where the project should be installed, and run ``git clone`` 
+* Switch to the newly installed project folder and run ``npm install``
+* If the test suite is also needed Jest will need to be installed, e.g. by running ``npm install -D jest``
```

Comment:
Since jest is already included in your dependencies it will automatically be installed, so you should just include the command to runt he tests in this file